static/app/components/arithmeticInput/grammar.pegjs

```diff
@@ -70,15 +70,15 @@ divide
 
 function_value "function"
   = function_name open_paren spaces function_args? spaces closed_paren {
-    return text();
+    return tc.tokenFunction(text(), location());
```

Comment:
Location is an object with the start&end locations where the token is eg.
`end: { offset: 37, line: 1, column: 38 }`